mail_send_restriction/readme/DESCRIPTION.rst

```diff
@@ -0,0 +1,2 @@
+This module enhances the 'mail' application's functionality, limiting the sending of email messages externally.
+External sending is permitted only for models included in the 'allow_send_model_ids' field of mail subtypes.
```

Comment:
```suggestion
This module limits the creation of non-internal messages in the chatter to certain "allowed" models.
This means that, upon installation of the module, users will get an error message when they try to create a message in the chatter, unless otherwise configured.
```